viewing_party/party.py

```diff
@@ -0,0 +1,135 @@
+def create_movie(title, genre, rating):
+    new_movie = {
+        "title" : title,
+        "genre" : genre,
+        "rating" : rating,
+    }
+
+    # != and "is not" do the same
+    if title != None and \
+        genre != None and \
+        rating != None:
+        return new_movie
+
+def add_to_watched(user_data, movie):
+    user_data["watched"] = [movie]
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"] = [movie]
+    return user_data
+
+def watch_movie(user_data, title):
+    for friend_watched in user_data["watchlist"]:
+        if friend_watched["title"] == title:
+            user_data["watchlist"].remove(friend_watched)
+            user_data["watched"].append(friend_watched)
+    return user_data
+
+def get_watched_avg_rating(user_data):
+    if len(user_data["watched"]) < 1:
+        return 0
+    else:
+        total_ratings = 0
+        for friend_watched in user_data["watched"]:
+            total_ratings += friend_watched["rating"]
+    return total_ratings/len(user_data["watched"])
```

Comment:
I would suggest not naming anything friend_watched here. In the README.Md for this function, we are calculating the average rating of all movies in the watched list .....iterating through each movie

```suggestion
def get_watched_avg_rating(user_data):
    if len(user_data["watched"]) == 0:
        return 0
    else:
        total_ratings = 0
        for movie in user_data["watched"]:
            total_ratings += movie["rating"]
    return total_ratings/len(user_data["watched"])
```